hkm/templates/hkm/views/search.html

```diff
@@ -17,7 +17,7 @@ <h1 class="banner__h1">{% trans 'Search pictures' %}</h1>
         <div class="search-form__content input-group">
           <input class="search-form__field form-control" placeholder="{% trans 'Search' %}" type="text" name="search" {% if search_term %} value="{{ search_term }}"{% endif %} />
           <span class="search-form__btn-container input-group-btn">
-            <button class="search-form__btn btn btn-default" type="submit"><img src="/static/hkm/img/suurennuslasi.png"></img></button>
+            <button aria-label="{% trans 'Search' %}" class="search-form__btn btn btn-default" type="submit"><img src="/static/hkm/img/suurennuslasi.png"></img></button>
```

Comment:
Maybe add an empty alt attribute on the image here to mark it as decorative.

Reply:
Added.